cmd/create/accountroles/cmd.go

```diff
@@ -329,7 +329,7 @@ func run(cmd *cobra.Command, argv []string) {
 		mode, err = interactive.GetOption(interactive.Input{
```

Comment:
@ciaranRoche I'd be tempted to roll this up into a common function seeing as it's essentially the same thing everywhere apart from the question to the user AFAICS